### DISCUSSION TITLE
can I use my own mysql database located in synology NAS to work with firefly iii in docker version?

### DISCUSSION THREAD

**Discussion by zhanghengtian** [firefly-iii/firefly-iii#5742] [Firefly III]
I have finished the firefly iii installation in docker and it works fine. 
What if I need to use my own mysql database located in Synology NAS (I don't want a seperate db container in docker), can I make that happen? 
I didn't find any answer from the documentation for this.

**Comment by JC5** [comment]
Yes you can. Configure Docker (Docker compose or command line) to use your mysql database. You must provide the (ip) address and the account details.

If the database is empty Firefly III will fill it automatically.

**Reply by JC5** [reply]
Sorry, I know very little about synology
